01-fundamentals/first_letter.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+def get_user_password():
+    password = input("Tell me your password: ")
+    if not password:
+        get_user_password()
+    return password
+
+def show_first_capital_letter(password):
+    first_capital_letter = get_first_letter_of(password).upper()
+    print("The first (capital) letter you entered was: " + first_capital_letter)
+
+def get_first_letter_of(text):
```

Comment:
For future reference, when your function is as trivial as this and it uses the built in functionality of Python, there's no actual need for a separate function. But leave at as is for now.